dbld/rules

```diff
@@ -189,7 +189,7 @@ shell-%: setup
 images: $(foreach image,$(IMAGES), image-$(image))
 image: image-$(DEFAULT_IMAGE)
 image-kira: image-ubuntu-focal
-image-devshell: image-ubuntu-focal
+image-devshell: image-tarball
```

Comment:
I think we could add `doxygen` and `docbook-xsl` to [devshell] in `package.manifest`.

The question is what is `distcheck`... if we think that this is something related to devshell (IMHO this is the case), then aforementioned packages should be part of devshell.
Or we can create a ci-shell, based on devshell - but I think devshell would be better.

After we solve this distcheck problem: what is the difference between tarball and devshell images?

Reply:
We discussed this with @bazsi and thought that basing `devshell`  on a rolling release would have advantages (should be same distro on which we do release tarball, which is debian testing).

Those packages should be part of devshell, and `distcheck` should run on `devshell` image, but they are installed on `tarball` images (which is an abstraction of debian testing).

`devshell` image doesn't have to be kept minimal contrary to `tarball`.

Reply:
we should run `make distcheck`  where we run `make dist`

Reply:
Agreed. This means that we need to move distcheck from this workflow to somewhere else. Right now it runs on devshell, but with the latest fixes that should work too.

However moving distcheck to somewhere else should be a separate PR.

Reply:
I've added this as a to-do: https://github.com/syslog-ng/syslog-ng/issues/3517